### PR TITLE
update to DW 1.0.0

### PR DESCRIPTION
**Pull request by qinfchen**

### CODE REVIEW

**Comment by qinfchen** [comment]
@zoubeiri

**Comment by qinfchen** [comment]
@jmcampanini can you review this one as well? we may need to wait for Merrick

**Comment by jmcampanini** [comment]
seems reasonable, but i don't have context on the actual logging bit. defer to @zoubeiri

**Comment by qinfchen** [comment]
@zoubeiri, can you review and maybe qe this?

**Comment by zoubeiri** [comment]
👍

**Comment by qinfchen** [comment]
how do we QE this?

File: src/main/java/com/palantir/weblogger/AnalyticsAppenderFactory.java [hunk 1]
```diff
@@ -59,27 +65,27 @@
         appender.setPrudent(false);
 
         LayoutWrappingEncoder<ILoggingEvent> layoutEncoder = new LayoutWrappingEncoder<>();
-        layoutEncoder.setLayout(layout == null ? buildLayout(context, getTimeZone()) : layout);
+        layoutEncoder.setLayout(buildLayout(context, layoutFactory));
         appender.setEncoder(layoutEncoder);
 
         // instead of adding a threshold filter, filter out all events not directed at the analytics logger
         appender.addFilter(new AnalyticsFilter());
-        appender.stop();
```
Comment (qinfchen) [inline src/main/java/com/palantir/weblogger/AnalyticsAppenderFactory.java:67]: @zoubeiri logback RollingFileAppender changed its impl, and `stop()` is not needed anymore.
https://github.com/qos-ch/logback/commit/1af14db4b6f28aee79f8f1bc866dfffe6cfdbe5a#diff-168f8cbb3bd8b03acc60eb147007d385R51

Comment (qinfchen) [reply src/main/java/com/palantir/weblogger/AnalyticsAppenderFactory.java:67]: actually, `stop` cannot be called, or `start` won't create the log file

**Pull request merged**